SDL2 panic: vert_shdr failed to compile

@strangebroadcasts we have a problem

macOS Sierra 10.12.1
go version go1.8 darwin/amd64

```
$ nk-example-sdl2

2017/03/28 13:38:58 SDL2: created window 400x500
panic: vert_shdr failed to compile

goroutine 1 [running, locked to thread]:
github.com/golang-ui/nuklear/nk.deviceCreate()
	/Users/xlab/Documents/dev/go/src/github.com/golang-ui/nuklear/nk/impl_sdl2_gl3.go:150 +0x580
github.com/golang-ui/nuklear/nk.NkPlatformInit(0x461e8f0, 0x47d20c0, 0x1, 0x2)
	/Users/xlab/Documents/dev/go/src/github.com/golang-ui/nuklear/nk/impl_sdl2_common.go:45 +0x91
main.main()
	/Users/xlab/Documents/dev/go/src/github.com/golang-ui/nuklear/cmd/nk-example-sdl2/main.go:49 +0x3e6
```

--- a/cmd/nk-example-sdl2/main.go
+++ b/cmd/nk-example-sdl2/main.go
@@ -33,6 +33,10 @@ func main() {
 	}
 	defer win.Destroy()
 
+	sdl.GL_SetAttribute(sdl.GL_CONTEXT_MAJOR_VERSION, 3)
+	sdl.GL_SetAttribute(sdl.GL_CONTEXT_MINOR_VERSION, 2)
+	sdl.GL_SetAttribute(sdl.GL_CONTEXT_PROFILE_MASK, sdl.GL_CONTEXT_PROFILE_CORE)
+
 	context, err := sdl.GL_CreateContext(win)
 	if err != nil {
 		closer.Fatalln(err)

